### PR TITLE
Docs restructuring

### PR DESCRIPTION
**Pull request by thejasn**
Signed-off-by: thejasn <thn@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# aws-load-balancer-operator
+# AWS LoadBalancer Operator
```
Comment (arjunrn) [inline README.md:1]:
```suggestion
# AWS Load Balancer Operator
```

File: docs/install.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# Using manual mode with STS
+# Installation
+
+This documents any required information either during installation or
+post installation to ensure the operator is successfully installed.
```
Comment (arjunrn) [inline docs/install.md:4]:
```suggestion
This documents any required information either during installation or
post installation to ensure the operator can function correctly.
```

File: docs/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# Pre-Requisites
+
+## CredentialsRequest
+
+### For non-STS clusters
+
+1. Additional credentials are needed for the operator to be successfully
```
Comment (arjunrn) [inline docs/prerequisites.md:7]:
```suggestion
1. Additional AWS credentials are needed for the operator to be successfully
```

File: docs/prerequisites.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    --from-file=credentials=credentials
+    ```
+  
+3. Alternatively use the *CredentialsRequest* resource in the `hack` directory
+
+   ```bash
+   oc apply -f https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
+   ```
+
+### Fos STS clusters
```
Comment (arjunrn) [inline docs/prerequisites.md:29]:
```suggestion
### For STS clusters
```

File: docs/prerequisites.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+The same steps mentioned above can be followed for STS clusters as well.
+But if instead of manually creating the secret in `step 2`, you prefer
+`step 3`, then there are additional steps involved.
+
+1. Post creating the *CredentialRequests*,
```
Comment (arjunrn) [inline docs/prerequisites.md:35]:
```suggestion
1. After creating the *CredentialRequests*,
```

File: docs/prerequisites.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+    ```bash
+    oc get credentialsrequest -n openshift-cloud-credential-operator  \
+        aws-load-balancer-controller -o yaml > <path-to-credrequests-dir>/cr.yaml
+    ```
+
+    Extract and save the required *CredentialsRequest* in a directory.
+
+2. Use the ccoctl tool to process all *CredentialsRequest* objects in the credrequests
+directory:
```
Comment (arjunrn) [inline docs/prerequisites.md:45]:
```suggestion
2. Use the ccoctl tool to process all *CredentialsRequest* objects in the previously specified
directory:
```

File: docs/prerequisites.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    of secrets in a **manifests** directory that is required
+    by the **aws-load-balancer-operator**.
+
+    **Note**: To Extract and prepare the `ccoctl` binary, documentation can be
+    found [here](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts).
+
+3. Apply the secrets to your cluster:
+
+    ```bash
+    ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
```
Comment (arjunrn) [inline docs/prerequisites.md:66]: Are there multiple secrets or just one?

**Review by arjunrn — COMMENTED** [review]
Looks good except for a couple of suggestions, thanks for making the docs nicer.

File: docs/prerequisites.md (hunk 7 above)
Comment (thejasn) [reply docs/prerequisites.md:66]: Just one, that is just the suffix it is generated with.

**Pull request merged**